src/server/qgsserversettings.cpp

```diff
@@ -615,3 +638,23 @@ QString QgsServerSettings::serviceUrl( const QString &service ) const
 
   return result;
 }
+
+int QgsServerSettings::projectCacheCheckInterval() const
+{
+  return value( QgsServerSettingsEnv::QGIS_SERVER_PROJECT_CACHE_CHECK_INTERVAL ).toInt();
+}
+
+QString QgsServerSettings::projectCacheStrategy() const
+{
+  QString result = value( QgsServerSettingsEnv::QGIS_SERVER_PROJECT_CACHE_STRATEGY ).toString();
+  if ( !result.compare( QLatin1String( "filesystem" ) ) ||
+       !result.compare( QLatin1String( "periodic" ) ) ||
+       !result.compare( QLatin1String( "off" ) ) )
```

Comment:
`QStringLiteral` instead of `QLatin1String`?

Reply:
If there is an overload of `QString::compare( QLatin1String )` the current approach is preferable.

Reply:
ok thanks :+1: